grunt/tasks/jasmine.js

```diff
@@ -9,7 +9,8 @@ var defaultOptions = {
     'node_modules/source-map-support/browser-source-map-support.js',
     'test/install-source-map-support.js',
     'http://maps.googleapis.com/maps/api/js?key=AIzaSyA4KzmztukvT7C49NSlzWkz75Xg3J_UyFI',
-    'node_modules/jasmine-ajax/lib/mock-ajax.js'
+    'node_modules/jasmine-ajax/lib/mock-ajax.js',
+    'https://raw.githubusercontent.com/CartoDB/tangram-1/point-experiment/dist/tangram.min.js'
```

Comment:
Yep, it is needed to load tangram.cartodb. Tangram needs to be loaded in a separate file without being part of a bundle.